ark/utils/notebooks_test.py

```diff
@@ -23,3 +37,234 @@ def test_example_spatial_analysis():
 
 def test_example_neighborhood_analysis():
     _exec_notebook('example_neighborhood_analysis_script.ipynb')
+
+
+# testing specific inputs for Segment_Image_Data
+def segment_notebook_setup(tb, deepcell_tiff_dir="test_tiff", deepcell_input_dir="test_input",
+                           deepcell_output_dir="test_output",
+                           single_cell_dir="test_single_cell",
+                           is_mibitiff=False, mibitiff_suffix="-MassCorrected-Filtered",
+                           num_fovs=3, num_chans=3, dtype=np.uint16):
+    # import modules and define file paths
+    tb.execute_cell('import')
+
+    # create the path to the directory containing the input data
+    tiff_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
```

Comment:
Is there a reason we need this complicated setup rather than overwriting the path like you defined above?

Reply:
I don't know why I did it this way anymore, can definitely make this simpler.

Reply:
@ngreenwald so it looks like the other reason we need to do this is because currently, `validate_paths` requires a `../data` prefix. Unfortunately, this means we need to hack this into the path when creating the `deepcell_tiff_dir` directory. Is there a reason we need this `../data` prefix still?

Reply:
@ngreenwald OK, so it looks like the reason we have to do it this way is because of the way `pytest` looks at the directory structure. This is the same reason why we have to set `path` in `exec_notebook` using `os.path.dirname(os.path.realpath(__file__))` as the root as well. I understand it's confusing though, because it looks like `testbook` has a different root than `pytest`.

One way around this is to use `tb.inject` to create `tiff_path` inside the notebook, but I don't think it's worth having to format a long string to do this. I'm fine with leaving `os.path.dirname(os.path.realpath(__file__))` in just that once, but if you see a reason we need to remove it then I can go with the `tb.inject` method instead.